{"uuid": "8527e2c3-1274-4582-b249-e775505534ea", "children": ["58dfcb35-a8f1-4b02-9946-286188171285"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741873413775, "stop": 1741873413775}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 101, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741873414092, "stop": 1741873414095}], "start": 1741873413775, "stop": 1741873414095}